app/views/claim-success.njk

```diff
@@ -12,17 +12,11 @@
     <p>Your claim has been successfully submitted.</p>
     <p>We've sent you a confirmation email.</p>
     <h2 class="govuk-heading-m">What happens next</h2>
-    <p>The Rural Payments Agency may contact you to request the review summary the vet has given you.</p>
+    <p>The Rural Payments Agency (RPA) may contact you to request the review summary the vet has given you.</p>
     <h2 class="govuk-heading-m">Get help with your claim</h2>
     <p>Contact the Rural Payments Agency if you need to change the details of your claim - for example if you've entered the vet's Royal College of Veterinary Surgeons number incorrectly.</p>
```

Comment:
This should be _RPA_ and not _Rural Payment Agency_

Reply:
hmmm later version prototype does not show that -> https://fcp-ahwr-prototype.herokuapp.com/v10/defraid-claim/defraid-1/confirmation

Reply:
Corrected!